src/errors/errorHandlers.ts

```diff
@@ -0,0 +1,49 @@
+import BaseError from './BaseError';
+import { BackendError, ForbiddenError, NetworkError, NotFoundError, ValidationError } from './network';
+import { ErrorInitializerFunction, NetworkErrorArgs } from './types';
+
+export const initializeError: ErrorInitializerFunction<NetworkError, NetworkErrorArgs> = (
+    ErrorType,
+    args
+): BaseError => {
+    let errorInstance: BaseError;
+    if (args.exception?.errors) {
+        const { exception, httpStatusCode } = args;
+        switch (httpStatusCode) {
+            case 400:
+                errorInstance =
+                    exception.title && (exception.title as string).toLowerCase().includes('validation')
+                        ? new ValidationError({ ...args, message: 'ValidationError' })
+                        : new BackendError({ ...args, message: 'BackendError' });
+                break;
+            case 403:
+                errorInstance = new ForbiddenError({ ...args, message: 'ForbiddenError' });
+                break;
+            case 404:
+                errorInstance = new NotFoundError({ ...args, message: 'NotFoundError' });
+                break;
+            default:
+                errorInstance = new BackendError({ ...args, message: 'BackendError' });
+                break;
+        }
+    } else {
+        errorInstance = new ErrorType({ args, message: 'NetworkError' });
+    }
+    return errorInstance;
+};
+
+export const handleClientError = (

```

Comment:
Bør renames til å convertToError eller noe sånt. Siden den ikke håndterer error lengre, men kun converter. Kan og være denne funksjonen ikke er nødvendig lengre?

Reply:
Jepp, det brukes som intermediate function som spesifiserer nødvnedige type, kan slettas uansett. Hva syns du, @Noggling ?

Reply:
Er ikke sikker, enig med @OveBDev uansett at den bør renames men kanskje vi kan legge den in under Utils eller noe. men initializeError gjør jo jobben. så mulig du bare kan slette den